core/docs/API.md

```diff
@@ -61,3 +61,21 @@
 
 11. Return details of source selected by id;<br>
     GET - https://ddw.devinit.org/api/sources/[SOURCE_ID]/
+
+12. Return list of Frozen Data
+    Get - https://ddw.devinit.org/api/frozendata/
+
+13. Return details of Frozen Data selected by ID
+    Get - https://ddw.devinit.org/api/frozendata/[FROZENDATA_ID]/
+
+12. Return list of Saved Query Data
+    Get - https://ddw.devinit.org/api/savedquerysets/
+
+13. Return details of Saved Query Data selected by ID
+    Get - https://ddw.devinit.org/api/savedqueryset/[SAVEDQUERYDATA_ID]/
+
+14. Download Frozen Data
+    Get - https://localhost/api/tables/download/[FROZEN_TABLE_NAME]/archives/
+
+15. Download Saved Query Set Data
+    Get - https://localhost/api/tables/download/[QUERY_SET_DB_TABLE]/dataset/
```

Comment:
Given that the initial context is the data source e.g. FTS ... how do I fetch only frozen datasets for a particular datasource?

Also 14 & 15 are linking to localhost ... ah, and the numbering is wrong.

Reply:
> Given that the initial context is the data source e.g. FTS ... how do I fetch only frozen datasets for a particular datasource?
> 

I worked on the assumption that you download a dataset one at a time, they can grow very huge! You will need the `frozen table name` and one of two schemas i.e dataset for saved querysets or archives for frozen data

> Also 14 & 15 are linking to localhost ... ah, and the numbering is wrong.

Localhost and numbering fixed.